### ISSUE TITLE
[Filter List Issues]: Searching on Bing shows blocked websites in the short videos section

### ISSUE THREAD

**Issue by smordarski** [Mordarski-Networks/Mordarski-Networks-assets#34]
I found out when you search on Bing, you will still see the blocked websites in the short videos section. I don't like short videos at all but for people that do but don't want to see a blocked website, I need to figure out how to remove them from Bing. I use Google but made this filter list issue to report that the filter list is broken on Bing.

![image](https://github.com/Mordarski-Networks/Mordarski-Networks-assets/assets/14831083/5ccbf43a-96a6-48c5-8b39-f5b1ee906d6d)

As you can see in the photo, it shows a blocked website (TikTok) only in the short videos section.

**Comment by MEP239** [comment]
wawa

**Comment by MEP239** [comment]
ill try to find something

**Comment by smordarski** [comment]
> wawa

Hello @MEP239, I use search results like "cat" or "Linux" for testing the filter list's functionality. Anyways, it is a bit odd that it is showing a blocked website on Bing. I will need to look into this more to figure out how to solve the problem. I am finally back after many real life stuff I had to do.

**Comment by MEP239** [comment]
Okay i dont know if im dumb... but the thing is that the short filter isnt working

**Comment by MEP239** [comment]
its alr

**Comment by MEP239** [comment]
> > wawa
> 
> Hello @MEP239, I use search results like "cat" or "Linux" for testing the filter list's functionality. Anyways, it is a bit odd that it is showing a blocked website on Bing. I will need to look into this more to figure out how to solve the problem. I am finally back after many real life stuff I had to do.

its alr

**Comment by smordarski** [comment]
> Okay i dont know if im dumb... but the thing is that the short filter isnt working

This only effects Bing. Google Search doesn't seem to have any issue with blocking websites in the short videos section. Short videos are basically YouTube shorts, Snapchat, Instagram Reels, etc. I don't use any of them but it is annoying to see them. I guess I could also block the whole short video section but that would have to be voted on.

**Comment by MEP239** [comment]
the lets block it thing shut down :p

**Comment by MEP239** [comment]
> > Okay I don't know if I'm dumb... but the thing is that the short filter isn't working
> 
> This only effects Bing. Google Search doesn't seem to have any issue with blocking websites in the short videos section. Short videos are basically YouTube shorts, Snapchat, Instagram Reels, etc. I don't use any of them but it is annoying to see them. I guess I could also block the whole short video section but that would have to be voted on.

it ded

**Comment by smordarski** [comment]
> > > Okay I don't know if I'm dumb... but the thing is that the short filter isn't working
> > 
> > 
> > This only effects Bing. Google Search doesn't seem to have any issue with blocking websites in the short videos section. Short videos are basically YouTube shorts, Snapchat, Instagram Reels, etc. I don't use any of them but it is annoying to see them. I guess I could also block the whole short video section but that would have to be voted on.
> 
> it going die on the end of June

Yeah, I know that and it is a sad time. My plan is to create a Python 3 script that generates filter lists and to fix up Bing search results.
